### ISSUE TITLE
Bastions at exactly 0,0 have group issues

### ISSUE THREAD

**Issue by wingzero54** [CivMC/Civ#137]
We're having issues with a bastion block on exactly 0,0 randomly switching groups. This has not happened anywhere except at 0,0 so assumptions are it's an issue with dividing by 0 or something

**Comment by MrJeremyFisher** [comment]
Is this still an issue?

**Comment by walkersgaming** [comment]
Yes, it isn't specific to bastions. Several blocks on x0 z0 have switched off our groups. cc @awoo-civ

**Comment by MrJeremyFisher** [comment]
Neat. Any additional info you have would be helpful as I haven't yet managed to reproduce it at all.
